.travis.yml

```diff
@@ -85,15 +85,8 @@ matrix:
    - env: GHCVER=8.0.2 SCRIPT=bootstrap
      os: osx
 
-   - env: GHCVER=via-stack SCRIPT=stack STACKAGE_RESOLVER=lts
+   - env: GHCVER=via-stack SCRIPT=stack STACK_CONFIG=stack.yaml
      os: linux
-     addons:
-        apt:
-            packages:
-                - libgmp-dev
-
-  allow_failures:
```

Comment:
Please leave `allow_failures` on, see [previous discussion](https://github.com/haskell/cabal/issues/4691#issuecomment-322471915) for rationale.